docs/src/main/paradox/google-fcm.md

```diff
@@ -0,0 +1,99 @@
+# Google Firebase Cloud Messaging
+
+The google firebase cloud messaging connector provides a way to send notifications https://firebase.google.com/docs/cloud-messaging/ .
```

Comment:
Make it a link instead.

Reply:
https://developer.lightbend.com/docs/alpakka/current/google-cloud-pub-sub.html
https://raw.githubusercontent.com/akka/alpakka/master/docs/src/main/paradox/google-cloud-pub-sub.md

It will be a link. (It's a link in local too.)